catalog/app/containers/Bucket/PackageDialog/FilesState.ts

```diff
@@ -0,0 +1,308 @@
+import { join, basename } from 'path'
+
+import * as R from 'ramda'
+import { FileWithPath } from 'react-dropzone'
+
+import type * as Model from 'model'
+import dissocBy from 'utils/dissocBy'
+import * as s3paths from 'utils/s3paths'
+import * as tagged from 'utils/taggedV2'
+import * as Types from 'utils/types'
+
+export const EMPTY_DIR_MARKER = '.quiltkeep'
```

Comment:
i think a key like this can be encountered in a real package, so i'd use something more obscure, preferably an invalid path

Reply:
I don't know what path would be invalid. Used this `[$.quiltkeep$]`